src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1941,6 +1942,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <TargetPlatformMonikers>$(TargetPlatformMoniker)</TargetPlatformMonikers>
         <TargetPlatformMonikers Condition="'$(TargetPlatformMoniker)' == ''">None</TargetPlatformMonikers>
         <AdditionalPropertiesFromProject>$(_AdditionalTargetFrameworkInfoProperties)</AdditionalPropertiesFromProject>
+
```

Comment:
Does _TargetFrameworkInfo ever have more than just $(TargetFramework) included in it?

Reply:
Yes, it does for multi-targeted projects, which will run this target for each target framework and combine the results.